app/Kconfig

```diff
@@ -522,6 +522,18 @@ choice CBPRINTF_IMPLEMENTATION
 
 endchoice
 
+DT_COMPAT_ZMK_BEHAVIOR_SENSOR_ROTATE := zmk,behavior-sensor-rotate
```

Comment:
This is already a pretty large file. Since we're likely to do this for all of the behaviors at some point, any thoughts on adding a src/behaviors/Kconfig (and maybe doing the same for cmake stuff)?

Reply:
Moved to `app/Kconfig.behaviors` for now. I think in the future, we'll:

* Move to `app/src/behaviors/Kconfig`
* Add `app/src/behaviors/CMakeLists.txt`
* Tweak the current conditions for split/central state for the behavior compilation in `app/CMakeLists.txt` into individual Kconfig symbols for each behavior that use Kconfig logic for conditionally enabling them or not.

Don't really want to cram all of that into this PR though.

Thoughts?

Reply:
Seems reasonable to me.